docs/authentication/session-management-console.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Admin Console actions to support the various development options
+id: session-management-console
+description: 'An overview of admin console configurations for session management within your application'
+slug: /session-management-console
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/15/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ThemedImage from '@theme/ThemedImage';
+
+## Admin Console
+
+In the Admin Console, there are a number of options for token configuration within your application.
+
+Note: Refresh tokens can only be configured and enabled for applications with an "Authorization Code" grant type.
+
+<Tabs>
+<TabItem value="client-credentials" label="Client Credentials">
+
+<ThemedImage
+  sources={{
+    light: '/img/docs/console-client-credentials-tokens-configuration.png',
+    dark: '/img/docs/console-client-credentials-tokens-configuration-dark.png',
+  }}
+/>
+
+</TabItem>
+<TabItem value="authorization-code" label="Authorization Code">
+
+<ThemedImage
+  sources={{
+    light: '/img/docs/console-authorization-code-tokens-configuration.png',
+    dark: '/img/docs/console-authorization-code-tokens-configuration-dark.png',
+  }}
+/>
+
+</TabItem>
+</Tabs>
+
+### Token Format
+
+There are two token formats that can be validated: `self-contained` or `referential`.
```

Comment:
I think it'd be more appropriate to preserve the exact string value that is used in the API:

```suggestion
There are two token formats that can be validated: `self_contained` or `referential`.
```